src/components/LobbyWrapper/LobbyWrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
I'd consider making these changes in a separate PR. It's making the App.tsx harder to diff.